OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu --> Demi_add_previous_page_redirect

# Description
Implements Phase 2 WBS Bugs line item 5 “Login”

## Related PRS (if any):
Use the development branch for the back end.

## Main changes explained:
- Redirects user to the previous page after log in on the BM Dashboard

## How to test:

1. pull the latest code
2. check into current branch
3. `npm install`
4. `npm run start:local` to run this PR locally
5. Clear site data/cache in local storage
6. log in as any user
7. go to a page of bmdashboard that requires authentication such as `http://localhost:3000/bmdashboard/materials`
8. since you are not authenticated, you will be redirect to bmdashboard/login
9. after successfully logged in, verify that you will be redirect to `bmdashboard/materials` (or any original page) page

## Screenshots or videos of changes:

https://www.loom.com/share/7ebc7400d6c8402d976ddd0b6038eeff?sid=0b40deae-2fdc-40d2-bf41-1bc6dd43476e 

## Note:
Accessing `bmdashboard/login` directly from URL will not trigger redirection.
